server/server.go

```diff
@@ -56,7 +56,7 @@ func (keys *defaultKeystore) Add(op *gokeyless.Operation, priv crypto.Signer) er
 	keys.Lock()
 	defer keys.Unlock()
 
-	if digest, ok := gokeyless.GetDigest(priv.Public()); ok {
+	if digest, err := gokeyless.GetDigest(priv.Public()); err == nil {
```

Comment:
It's sort of implicit, just the same as all the following optional map keys. But, sure.